components/html-block/html-block.js

```diff
@@ -194,7 +206,12 @@ class HtmlBlock extends RtlMixin(LitElement) {
 		super.firstUpdated(changedProperties);
 
 		if (this._renderContainer) return;
-		this.shadowRoot.innerHTML += `<div class="d2l-html-block-rendered${this.compact ? ' d2l-html-block-compact' : ''}"></div><slot ${!this.noDeferredRendering ? 'style="display: none"' : ''}></slot>`;
+		this.shadowRoot.innerHTML += '<div class="d2l-html-block-rendered'
+			+ `${this.compact ? ' d2l-html-block-compact' : ''}`
+			+ `${this.noVerticalMargins ? ' d2l-html-block-no-vertical-margins' : ''}`
+			+ '"></div><slot'
+			+ `${!this.noDeferredRendering ? ' style="display: none"' : ''}`
+			+ '></slot>';
```

Comment:
Formatting because the one line was starting to get ridiculously long. I still want to spend time investigating whether we can get Lit rendering to play nicely with MathJax, but... That's for another time.